flexget/plugins/filter/duplicates.py

```diff
@@ -16,32 +16,82 @@ class Duplicates(object):
       duplicates: 
         field: <field name>
         action: [accept|reject]
+
+    Reject the second+ instance of every movie:
+
+      duplicates:
+        field:
+          - imdb_id
+          - movie_name
+        action: reject
+        first: true # Allow first instance
     """
 
     schema = {
         'type': 'object',
         'properties': {
-            'field': {'type': 'string'},
+            'field': {
+                'oneOf': [
+                    {'type': 'string'},
+                    {'type': 'array'},
+                ]
+            },
             'action': {'enum': ['accept', 'reject']},
+            'first': {'type': 'boolean'},
         },
         'required': ['field', 'action'],
         'additionalProperties': False
     }
 
+    def prepare_config(self, config):
+        if config is None:
+            config = {}
+        config.setdefault('first', True)
+        return config
+
+    def extract_fields(self, entry, field_names):
+        ret = []
+        for field in field_names:
+            if field in entry:
+                ret.append(entry[field])
+            else:
+                ret.append(None)
+        return ret
+
     def on_task_filter(self, task, config):
-        field = config['field']
+        config = self.prepare_config(config)
+        field_names = config['field']
+        # Convert to list
+        if not isinstance(field_names, list):
+            field_names = [field_names]
         action = config['action']
-        for entry in task.entries:
-            for prospect in task.entries:
-                if entry == prospect:
-                    continue
-                if entry[field] == prospect[field] and entry[field] is not None:
+        entries = list(task.entries)
+        entry_len = len(entries)
+        for i in range(entry_len):
```

Comment:
I have no answer to that. What are the benefits of it?
I only moved to list because I had some weird behavior when directly accessing items (`task.entries[i]`)